Add ErrorReasons to 'core-action-status-get' Command (#37483)
demisto/content: Packs/CortexXDR/Integrations/CortexXDRIR/README.md

| incident_id | Allows linking the response action to the incident that triggered it. | Optional | 
| endpoint_id_list | List of endpoint IDs. | Required | 
| file_path | String that represents the path of the file you want to quarantine. | Required | 
| file_hash | String that represents the file's hash. Must be a valid SHA256 hash. | Required | 
| interval_in_seconds | Interval in seconds between each poll. | Optional | 
| timeout_in_seconds | Polling timeout in seconds. | Optional | 
| action_id | The action IDs for polling use. | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |

> | PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket that the error happened in. | 

Reviewer: ```suggestion
| PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket in which the error occurred. | 
```

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| incident_id | Allows linking the response action to the incident that triggered it. | Optional | 
| file_hash | The hash code of the file. Must be a valid SHA256 hash. | Required | 
| endpoint_id | String that represents the endpoint ID. If you do not enter a specific endpoint ID, the request will run restore on all endpoints that relate to the quarantined file you defined. | Optional | 
| interval_in_seconds | Interval in seconds between each poll. | Optional | 
| timeout_in_seconds | Polling timeout in seconds. | Optional | 
| action_id | The action IDs for polling use. | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |

> | PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket that the error happened in. | 

Reviewer: ```suggestion
| PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket in which the error occurred. | 
```

| lte_last_seen | Less than or equal to last seen timestamp in milliseconds. | Optional | 
| ip_list | List of IP addresses. | Optional | 
| group_name | Name of the endpoint group. | Optional | 
| platform | Type of operating system. Possible values are: windows, linux, macos, android. | Optional | 
| alias | Endpoint alias name. | Optional | 
| isolate | Whether an endpoint has been isolated. Possible values are: isolated, unisolated. | Optional | 
| hostname | Name of the host. | Optional | 
| all | Whether to scan all of the endpoints. Scanning all of the endpoints may cause performance issues and latency. Possible values are: true, false. Default is false. | Optional | 
| action_id | The action IDs for polling use. | Optional | 
| interval_in_seconds | Interval in seconds between each poll. | Optional | 
| timeout_in_seconds | Polling timeout in seconds. | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| PaloAltoNetworksXDR.endpointScan.actionId | Number | The action ID of the scan request. | 
| PaloAltoNetworksXDR.endpointScan.aborted | Boolean | Was the scan aborted? | 

> | PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket that the error happened in. | 

Reviewer: ```suggestion
| PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket in which the error occurred. | 
```

| PaloAltoNetworksXDR.RetrievedFiles.endpoint_id | string | Endpoint ID. Added only when the operation is successful. | 
| PaloAltoNetworksXDR.RetrievedFiles.file_link | string | Link to the file. Added only when the operation is successful. | 
| PaloAltoNetworksXDR.RetrievedFiles.status | string | The action status. Added only when the operation is unsuccessful. | 

> | PaloAltoNetworksXDR.RetrievedFiles.ErrorReasons.bucket | String | The bucket that the error happened in. | 

Reviewer: ```suggestion
| PaloAltoNetworksXDR.RetrievedFiles.ErrorReasons.bucket | String | The bucket in which the error occurred. | 
```

| --- | --- | --- |
| incident_id | Allows linking the response action to the incident that triggered it. | Optional | 
| endpoint_ids | A comma-separated list of endpoint IDs. Can be retrieved by running the xdr-get-endpoints command. | Required | 
| snippet_code | Section of a script to initiate on an endpoint (e.g., print("7")). | Required | 
| interval_in_seconds | Interval in seconds between each poll. | Optional | 
| timeout_in_seconds | Polling timeout in seconds. | Optional | 
| action_id | Action IDs for polling use. | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| PaloAltoNetworksXDR.ScriptRun.action_id | Number | ID of the action initiated. | 
| PaloAltoNetworksXDR.ScriptRun.endpoints_count | Number | Number of endpoints the action was initiated on. | 

> | PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket that the error happened in. | 

Reviewer: ```suggestion
| PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket in which the error occurred. | 
```

| commands | A comma-separated list of shell commands to execute. Set the `is_raw_command` argument to `true` to prevent splitting by commas. (Useful when using `\|\|`, `&amp;&amp;`, `;` separators for controlling the flow of multiple commands). | Required | 
| is_raw_command | Whether to pass the command as-is. When false, the command is split by commas and sent as a list of commands, that are run independently. | Optional | 
| command_type | Type of shell command. Possible values are: powershell, native. | Optional | 
| timeout | The timeout in seconds for this execution. Default is 600. | Optional | 
| interval_in_seconds | Interval in seconds between each poll. | Optional | 
| timeout_in_seconds | Polling timeout in seconds. | Optional | 
| action_id | The action IDs for polling use. | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| PaloAltoNetworksXDR.ScriptRun.action_id | Number | ID of the action initiated. | 
| PaloAltoNetworksXDR.ScriptRun.endpoints_count | Number | Number of endpoints the action was initiated on. | 

> | PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket that the error happened in. | 

Reviewer: ```suggestion
| PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket in which the error occurred. | 
```

| endpoint_ids | A comma-separated list of endpoint IDs. Can be retrieved by running the xdr-get-endpoints command.                    | Required | 
| file_path | A comma-separated list of paths of the files to delete. All of the given file paths will run on all of the endpoints. | Required | 
| timeout | The timeout in seconds for this execution. Default is 600.                                                            | Optional | 
| interval_in_seconds | Interval in seconds between each poll.                                                                                | Optional | 
| timeout_in_seconds | Polling timeout in seconds.                                                                                           | Optional | 
| action_id | The action IDs for polling use.                                                                                       | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| PaloAltoNetworksXDR.ScriptRun.action_id | Number | ID of the action initiated. | 
| PaloAltoNetworksXDR.ScriptRun.endpoints_count | Number | Number of endpoints the action was initiated on. | 

> | PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket that the error happened in. | 

Reviewer: ```suggestion
| PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket in which the error occurred. | 
```

| timeout | The timeout in seconds for this execution. Default is 600. | Optional | 
| interval_in_seconds | Interval in seconds between each poll. | Optional | 
| timeout_in_seconds | Polling timeout in seconds. | Optional | 
| action_id | The action IDs for polling use. | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| PaloAltoNetworksXDR.ScriptRun.action_id | Number | ID of the action initiated. | 
| PaloAltoNetworksXDR.ScriptRun.endpoints_count | Number | Number of endpoints the action was initiated on. | 

> | PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket that the error happened in. | 

Reviewer: ```suggestion
| PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket in which the error occurred. | 
```

| timeout | The timeout in seconds for this execution. Default is 600. | Optional | 
| interval_in_seconds | Interval in seconds between each poll. | Optional | 
| timeout_in_seconds | Polling timeout in seconds. | Optional | 
| action_id | The action IDs for polling use. | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| PaloAltoNetworksXDR.ScriptRun.action_id | Number | ID of the action initiated. | 
| PaloAltoNetworksXDR.ScriptRun.endpoints_count | Number | Number of endpoints the action was initiated on. | 

> | PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket that the error happened in. | 

Reviewer: ```suggestion
| PaloAltoNetworksXDR.GetActionStatus.ErrorReasons.bucket | String | The bucket in which the error occurred. | 
```
